GVFS/GVFS.Common/ModifiedPathsDatabase.cs

```diff
@@ -52,6 +52,34 @@ public static bool TryLoadOrCreate(ITracer tracer, string dataDirectory, Physica
             return true;
         }
 
+        public void Compress(ITracer tracer)

```

Comment:
How long does this take with a large number of entries?

Reply:
```
[9/25/2018 6:16:15 PM] Compress ModifiedPaths (Stop) {"startingCount":1839665,"EndCount":9528,"DurationMs":1451}
```